runelite-client/src/main/java/net/runelite/client/plugins/cannon/CannonConfig.java

```diff
@@ -25,28 +25,43 @@
 package net.runelite.client.plugins.cannon;
 
 import java.awt.Color;
-import net.runelite.client.config.Alpha;
-import net.runelite.client.config.Config;
-import net.runelite.client.config.ConfigGroup;
-import net.runelite.client.config.ConfigItem;
+
+import net.runelite.client.config.*;
 
 @ConfigGroup("cannon")
 public interface CannonConfig extends Config
 {
 	@ConfigItem(
-		keyName = "showEmptyCannonNotification",
-		name = "Empty cannon notification",
-		description = "Configures whether to notify you that the cannon is empty"
+		keyName = "enableCannonNotifications",
+		name = "Enable cannon notifications",
+		description = "Configures whether to notify you when your cannon has X cannonballs remaining.",
+		position = 1
 	)
-	default boolean showEmptyCannonNotification()
+	default boolean enableCannonNotifications()
 	{
 		return true;
 	}
 
+	@Range(
+		max = 30
+	)
+	@ConfigItem(
+		keyName = "numRemainingBeforeNotification",
+		name = "Number for notification",
+		description = "Configures the of cannonballs remaining before the user should be notified. Regardless of this" +
```

Comment:
Typo. 
I'm guessing it's supposed to be `Configures the number of cannonballs`

Reply:
thanks - fixed